modules/GOTOPGO.nf

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env nextflow
+
+nextflow.preview.dsl=2
+
+process goTopGo {
```

Comment:
You need to add the R conda envs here: e.g.
`conda "conda-forge::r-base=4.1.2 bioconda::bioconductor-topgo bioconda::bioconductor-org.hs.eg.db conda-forge::r-ggplot2"`